vnext/Microsoft.ReactNative.sln

```diff
@@ -133,12 +135,12 @@ Global
 		Shared\Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
```

Comment:
VS uses the first in the list, but writes alphabetic order...
Since we usually want to run x64 we should manually maintain X64 at the top of the list...

Reply:
Good point! I will revert this change. I had to do it in another PR yesterday per Andrew Coates feedback too.